src/api/api_routes.py

```diff
@@ -0,0 +1,68 @@
+# Non-public API. Don't host publicly - SECURITY RISKS!
+# (will only be on with --api starting option)
+# Currently no API stability guarantees are provided - API may break on any new commit.
+
+from fastapi import FastAPI, Body
+from fastapi.exceptions import HTTPException
+from itertools import tee
+
+import gradio as gr
+
+from typing import Dict, List
+
+from src.common_constants import GenerationOptions as go
+from src.misc import SCRIPT_VERSION
+from src.api.api_constants import api_options, models_to_index
+from src.api.api_core import api_gen, encode_to_base64
+
+# gr.Blocks is needed for auto1111 extensions
+def depth_api(_: gr.Blocks, app: FastAPI):
```

Comment:
Blocks object goes into memory hole right after it is supplied. I assume nothing depends on it. It should be removed (unnecessary argument).

Reply:
It's a type-hint so it shouldn't be given memory. 

It can be changed to 

`def depth_api(_, app: FastAPI):` or maybe `def depth_api(_grBlocks, app: FastAPI):` to help with context?

This function is being given to auto1111 so the unused argument can't be removed.